packages/core/cache/src/LMDBCache.js

```diff
@@ -73,12 +81,41 @@ export class LMDBCache implements Cache {
   }
 
   async setBlob(key: string, contents: Buffer | string): Promise<void> {
+    invariant(
+      !isLargeBlob(contents),
+      'Cannot store large blobs in the cache. You may want to use `setLargeBlob` instead.',
+    );
```

Comment:
Maybe we won't benefit from this check any more, since our most likely sources of large blobs are either being stored explicitly (graphs), or are being streamed (assets).